<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LauncherWindow</class>
 <widget class="QMainWindow" name="LauncherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>470</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Ttyh Launcher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/favicon.png</normaloff>:/resources/favicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="logDisplay">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string notr="true"/>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextBrowserInteraction</set>
      </property>
      <property name="maximumBlockCount">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="logo">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/resources/logo.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="runPanel" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>4</number>
         </property>
         <item row="3" column="0" colspan="2">
          <widget class="QCheckBox" name="savePassword">
           <property name="text">
            <string>Remember password</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="nickLabel">
           <property name="text">
            <string>Player Name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="passLabel">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="clienttypeLabel">
           <property name="text">
            <string>Clinet</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QComboBox" name="clientCombo"/>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="passEdit">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="nickEdit"/>
         </item>
         <item row="3" column="2">
          <layout class="QHBoxLayout" name="playbuttonLayout">
           <item>
            <spacer name="btnSpc">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="playButton">
             <property name="text">
              <string>Play!</string>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
             <property name="default">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="optionsMenu">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="playOffline"/>
    <addaction name="hideLauncher"/>
    <addaction name="loadNews"/>
   </widget>
   <widget class="QMenu" name="addMenu">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="runSettings"/>
    <addaction name="changeSkin"/>
    <addaction name="updateManager"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="bugReport"/>
    <addaction name="aboutLauncher"/>
   </widget>
   <addaction name="optionsMenu"/>
   <addaction name="addMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <action name="playOffline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Play offline</string>
   </property>
  </action>
  <action name="runSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/configure.png</normaloff>:/resources/configure.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Client settings</string>
   </property>
  </action>
  <action name="bugReport">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/reporting.png</normaloff>:/resources/reporting.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Report a bug</string>
   </property>
  </action>
  <action name="changePassword">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/password.png</normaloff>:/resources/password.png</iconset>
   </property>
   <property name="text">
    <string>Изменить пароль</string>
   </property>
  </action>
  <action name="changeSkin">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/skin.png</normaloff>:/resources/skin.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Upload skin</string>
   </property>
  </action>
  <action name="updateManager">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/minecraft.png</normaloff>:/resources/minecraft.png</iconset>
   </property>
   <property name="text">
    <string>Update &amp;manager</string>
   </property>
  </action>
  <action name="aboutLauncher">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/about.png</normaloff>:/resources/about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="doClone">
   <property name="text">
    <string>&amp;Clone</string>
   </property>
  </action>
  <action name="doFetch">
   <property name="text">
    <string>&amp;Fetch</string>
   </property>
  </action>
  <action name="doCheckout">
   <property name="text">
    <string>C&amp;hecksumms</string>
   </property>
  </action>
  <action name="doExport">
   <property name="text">
    <string>&amp;Export</string>
   </property>
  </action>
  <action name="hideLauncher">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Hide launcher window</string>
   </property>
  </action>
  <action name="loadNews">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Load news</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>playButton</tabstop>
  <tabstop>nickEdit</tabstop>
  <tabstop>passEdit</tabstop>
  <tabstop>clientCombo</tabstop>
  <tabstop>savePassword</tabstop>
  <tabstop>logDisplay</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
